.docker/ubuntu-24.04/Dockerfile

```diff
@@ -0,0 +1,108 @@
+# Stage 1: Base image                                                                                                                                                                                                                                                                                                                                    
```

Comment:
should we push this to a separate PR first, so we can then use this PR to validate it?